app/controllers/TimeoutController.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.action.Actions
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.Views
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class TimeoutController @Inject() (actions: Actions,
+                                   views:   Views,
+                                   mcc:     MessagesControllerComponents)
+  (implicit ec: ExecutionContext) extends FrontendController(mcc) {
+  implicit def toFuture(r: Result): Future[Result] = Future.successful(r)
+
+  def keepAliveSession(): Action[AnyContent] = Action(NoContent)
```

Comment:
We may need to consider the Time-To-Live of the data held in the mongo DB here.

e.g. if the TTL is set to 30 mins, it might be that the user keeps the session alive for longer before moving on in their journey, at which point their journey is no longer in the DB since mongo cleared it out when the TTL expired.

Reply:
Ref: the TTL index ("expireAfterSeconds") in the `JourneyRepo` is on the `createdOnTime` field so that field may need updating in order to keep the journey data alive.

Reply:
Journey TTL is set to 366 days, made comment in application.conf regarding necessary TTL